src/madhuraMJun20/SumOfNumbersInString.java

```diff
@@ -0,0 +1,25 @@
+package madhuraMJun20;
+
+/*WAP to sum all the numbers in given string.
+Input: J7yu9y8h1h8j4b7j3jjb6
+Output: Sum of all numbers in String is: 53
+*/
+public class SumOfNumbersInString {
+
+	void sumOfNumber(String input) {
+		int sum = 0;
+		for (int index = 0; index < input.length(); index++) {
+			if (Character.isDigit(input.charAt(index))) {
+				int number = Integer.parseInt(String.valueOf(input.charAt(index)));
```

Comment:
You can use below method to convert char to int
Character.getNumericValue(numberString.charAt(index));